### PR TITLE
April community meeting recap blog

### PR DESCRIPTION
**Pull request by PaulMichaelArmstrong**
Signed-off-by: Paul Armstrong <paul@buoyant.io>

### CODE REVIEW

File: linkerd.io/content/blog/april-linkerd-community-meeting.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+date = 2021-05-14T00:00:00Z
+feature = "/uploads/april-community-meeting-recap_thumb.png"
+tags = ["Linkerd"]
+thumbnail = "/uploads/april-community-meeting-recap_thumb.png"
+title = "April Linkerd Community Meeting Recap"
+description = "We chatted with Fredrik Klingenberg about his experience implementing Linkerd at Elkjøp, talked about our roadmap, and much more."
+featured = false
++++
+
+As always, if you missed the April Linkerd Community Meeting, there is no reason to panic —  we've got you covered. Check out our recap and the recording.
```
Comment (wmorgan) [inline linkerd.io/content/blog/april-linkerd-community-meeting.md:12]: Maybe "Here's our" instead of "Check out". The current text makes me think I'm supposed to click on a link or something.

File: linkerd.io/content/blog/april-linkerd-community-meeting.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+Before getting started, we wanted to remind you of the [Linkerd Anchor program](https://linkerd.io/community/anchor/). If you have a great Linkerd story, we'd love to help you tell it. Whether assisting with the writing process, reviewing code, or providing guidance on how to create a great video tutorial —  our team is here to help. [Submit the form](https://linkerd.io/community/anchor/) and we'll reach out!
+
+We are also still gathering responses to the [Linkerd 2021 user survey](https://docs.google.com/forms/d/e/1FAIpQLSfofwKQDOrAN9E9Vg1041623A3-8nmEAxlAbvXw-S9r3QnT9g/viewform). If you haven't participated yet, please do so —  it really helps us make this awesome service mesh even more awesome.
+
+## Updates
+
+As mentioned last time, we’ve submitted Linkerd to graduation and have gotten lots of thumbs up – looking really good so far. We'll continue to keep you updated as things evolve. Linkerd also completed fuzz tests for the proxy. A blog post with details will follow soon.
```
Comment (wmorgan) [inline linkerd.io/content/blog/april-linkerd-community-meeting.md:20]: Fuzz testing blog post is live so let's link to it.

I also find this section a bit weird. It reads like a transcript, but the rest of the doc reads like a recap (which is what I would expect.) I would reword this as:

## Project updates

We've submitted Linkerd for CNCF graduation, and are currently waiting for a TOC sponsor. We've also added fuzz tests to the proxy, sponsored by the CNCF. You can read more in the blog post: [Introducing fuzz testing for Linkerd](https://linkerd.io/2021/05/07/fuzz-testing-for-linkerd/)

Comment (hawkw) [inline linkerd.io/content/blog/april-linkerd-community-meeting.md:20]: didn't we already publish the fuzz testing blog post? https://linkerd.io/2021/05/07/fuzz-testing-for-linkerd/

File: linkerd.io/content/blog/april-linkerd-community-meeting.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+  * To allow more circuit breaker options
+  * Support for fault injection
+* **Better gRPC support**:
+  * Enabling gRPC retries
+  * Broader enhancements for gRPC traffic with Linkerd
+* Better **integration with Open Metrics and Open Telemetry** as those APIs evolve
+
+## Linkerd @ KubeCon EU
+
+This KubeCon and CloudNativeCon were the most [Linkerd-heavy to date](https://buoyant.io/2021/05/13/kubecon-eu-2021-wrap-up/). We were particularly excited to see so many end users sharing their service mesh journey — it's just great to see how the community shares their lessons learned! This, by the way, ties directly into the [Linkerd Anchor Program](https://linkerd.io/community/anchor/). If you'd like to tell your story at KubeCon or at any other conference and need help, we'd love to assist you!
```
Comment (wmorgan) [inline linkerd.io/content/blog/april-linkerd-community-meeting.md:40]: I love m-dashes but stylistically speaking, there is one in almost every single paragraph. Let's replace some of them with good ol' fashioned periods.

**Comment by PaulMichaelArmstrong** [comment]
Thanks, @hawkw and @wmorgan. Here is the updated version.

**Review by wmorgan — APPROVED** [review]
Looks great. Thanks @PaulMichaelArmstrong
